### PR TITLE
Form refactor progress

### PR DESCRIPTION
**Pull request by styvone**
Integrated Michael's form refactor w/ Bryan's styling

### CODE REVIEW

File: src/components/FormItem/index.js [hunk 1]
```diff
@@ -38,147 +38,148 @@ class FormItem extends React.Component {
...
 
 FormItem.propTypes = {
-	entry: PropTypes.shape({
-		type: PropTypes.string.isRequired,
-		title: PropTypes.string.isRequired,
-		required: PropTypes.bool,
-		options: PropTypes.arrayOf(PropTypes.string),
-		secure: PropTypes.bool
-	})
+  entry: PropTypes.shape({
```
Comment (xualexander) [inline src/components/FormItem/index.js]: is the entry container necessary?

Comment (styvone) [reply src/components/FormItem/index.js]: "it's cleaner dud" - @michaelnyu

Comment (styvone) [reply src/components/FormItem/index.js]: "trust" - @michaelnyu

File: src/components/Form/index.js [hunk 2]
```diff
@@ -12,39 +12,35 @@ import PropTypes from 'prop-types';
...
 
-	handleSubmit() {
-		const {onSubmit} = this.props;
-		if (typeof onSubmit !== "undefined") {
-			onSubmit();
-		}
-	}
+  handleSubmit() {
+    const { onSubmit } = this.props;
+    if (typeof onSubmit !== "undefined") {
```
[the remark this answers is not in the document]
Comment (michaelnyu) [reply src/components/Form/index.js]: agree

**Pull request merged**